Velocity/about.cpp

```diff
@@ -9,22 +9,6 @@ About::About(QWidget *parent) :
     setFixedSize(sizeHint());
     setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
     QtHelpers::GenAdjustWidgetAppearanceToOS(this);
-
-#ifdef _WIN32
-    if (!QFile::exists(QtHelpers::ExecutingDirectory() + "Developers.mp3"))
```

Comment:
Why remove this? Would prefer to keep it, it's a nice easter egg

Reply:
I'm generally in favor of removing dependencies if this allows us to remove Phonon. Maybe we can use `QMediaPlayer` to play it?